scss/_pagination.scss

```diff
@@ -104,7 +104,7 @@
     margin-left: $pagination-margin-start;
   }
 
-  @if $pagination-margin-start == (calc($pagination-border-width * -1)) {
+  @if $pagination-margin-start == (-$pagination-border-width) {
```

Comment:
At first, I thought it could be easier to understand but it'll be harder to maintain it so let's keep it as-is to be like Bootstrap.